packages/cli/lib/validators/marketplaceValidators/theme/TemplateValidator.js

```diff
@@ -99,24 +100,23 @@ class TemplateValidator extends BaseValidator {
 
     files.forEach(file => {
       if (isCodedFile(file)) {
-        const annotations = getFileAnnotations(file);
+        let data;
+        try {
+          data = fs.readFileSync(file, 'utf8');
```

Comment:
Moves the file reading step from cli-lib to cli which I don't think has any negative impact